mix.exs

```diff
@@ -42,8 +42,7 @@ defmodule GoogleApis.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:tesla, "~> 0.5"},
```

Comment:
I think it's still good practice to declare the tesla dependency explicitly.

Reply:
done